tests/test_datasources.py

```diff
@@ -61,6 +61,9 @@ def test_print_data_sources(allkindofcomplexity, server_type):
 
 
 def test_data_sources_from_data_sources(allkindofcomplexity, server_type):
+    with pytest.raises(ValueError) as e:
+        data_sources_false = dpf.core.DataSources(data_sources="Wrong Input", server=server_type)
```

Comment:
@MichaelNale FYI, since I saw Codacy complaining, the pythonic thing to do here instead of having an unused variable is:
```suggestion
        _ = dpf.core.DataSources(data_sources="Wrong Input", server=server_type)
```